libs/s25main/desktops/dskGameLobby.cpp

```diff
@@ -99,6 +100,13 @@ dskGameLobby::dskGameLobby(ServerType serverType, std::shared_ptr<GameLobby> gam
               gameLobby_->isHost()); // This should be done first for the host so others won't even see the script
             LOG.write(_("Lua was disabled by the script itself\n"));
             lua.reset();
+        } else
+        {
+            if(const auto num = lua->GetNumPlayersFromScript())
+            {
+                GAMESERVER.SetNumPlayers(num);
```

Comment:
This won't work for savegames, will it? Because I think for a savegame the player needs to be already initialized.  
Or does that work still?